projects/ad400x/zed/system_bd.tcl

```diff
@@ -0,0 +1,9 @@
+
+source $ad_hdl_dir/projects/common/zed/zed_system_bd.tcl
+
+# specify ADC resolution -- supported resolutions 16/18/20 bits
+
+set adc_resolution 20
```

Comment:
I think it would be good to be able to specify the sample rate in MSPS a similar way. And then compute tcyc automatically, that should make it easier to adopt the project to the different parts in this ADC family.

Reply:
Yes, agreed. I can do that. Thanks!